locale/en/submission.po

```diff
@@ -2425,3 +2425,78 @@ msgstr "Competing Interests"
 
 msgid "author.competingInterests.description"
 msgstr "Please disclose any competing interests this author may have with the research subject."
+
+msgid "submission.list.reviewAssignment.statusAwaitingResponse.title"
+msgstr "Awaiting Response from the Reviewer"
+
+msgid "submission.list.reviewAssignment.statusAwaitingResponse.description"
+msgstr "Review request has been shared with Reviewer. Response is awaited in <b>{$days} days on {$date}</b>"
```

Comment:
Not the native speaker but think we can avoid passive voice here by changing  `Response is awaited in` to the `Awaiting response in` :)

Reply:
@Devika008 Any opinion here? Also should I change 'Reviewer' to 'reviewer'?